README.md

```diff
@@ -151,25 +155,26 @@ After it finishes, you will find `table5.txt`, and `table6.txt`, and `table7.txt
 
 The `table5.txt` should look like below:
 
-```
-Operator                           Undesired State    System Error    Operator Error    Recovery Failure    Total
--------------------------------  -----------------  --------------  ----------------  ------------------  -------
-CassOp                                           2               0                 0                   2        4
-CockroachOp                                      3               0                 2                   0        5
-KnativeOp                                        1               0                 2                   0        3
-OCK/RedisOp                                      4               1                 3                   1        9
-OFC/MongoOp                                      3               1                 2                   2        8
-PCN/MongoOp                                      4               0                 0                   1        5
-RabbitMQOp                                       3               0                 0                   0        3
-SAH/RedisOp                                      2               0                 0                   1        3
-TiDBOp                                           2               1                 0                   1        4
-XtraDBOp                                         4               0                 1                   1        6
-ZooKeeperOp                                      4               1                 0                   1        6
-Total                                           32               4                10                  10       56
+```text
+Operator         Undesired State    System Error    Operator Error    Recovery Failure    Total
+-------------  -----------------  --------------  ----------------  ------------------  -------
+CassOp                         2               0                 0                   2        4
+CockroachOp                    3               0                 2                   0        5
+KnativeOp                      1               0                 2                   0        3
+OCK-RedisOp                    4               1                 3                   1        9
+OFC-MongoDBOp                  3               1                 2                   2        8
+PCN-MongoDBOp                  4               0                 0                   1        5
+RabbitMQOp                     3               0                 0                   0        3
+SAH-RedisOp                    2               0                 0                   1        3
+TiDBOp                         2               1                 0                   1        4
+XtraDBOp                       4               0                 1                   1        6
+ZookeeperOp                    4               1                 0                   1        6
```

Comment:
Note: the results are slightly different as for the first column (operator name), but not the numbers. Looks like mine is consistent with the latest code of `sosp-ae`